airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -174,6 +174,10 @@ def transform_postgres(config: Dict[str, Any]):
             "threads": 32,
         }
 
+        # if unset, we assume true.
+        if config.get("ssl", True):
+            config["sslmode"] = "require"
```

Comment:
From DBT docs:
https://docs.getdbt.com/reference/warehouse-profiles/postgres-profile#sslmode

>  When unset, dbt will connect to databases using the Postgres default, prefer, as the sslmode.

Shouldn't we explicitly set it to "require" if it is unset in the destination config then?

```suggestion
        if "ssl" not in config or config.get("ssl", True):
            config["sslmode"] = "require"
```

Reply:
I just ran /test commands and they work now!

We're just waiting for an answer from this question before publishing then

Reply:
discussed offline. decided the original syntax is okay. if "config" is not set then get will default to true.